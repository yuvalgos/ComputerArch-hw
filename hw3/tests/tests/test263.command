./dflow_calc tests/test263.dat tests/test263.in p131 p126 p157 p6 p142 p93 d104 d37 d4 p116 p28 p63 p90 p91 d49 d39 p42 p144 d155 p23 p102 p9 p43 d79 d45 p162 d112 d85 d132 d26 d30 p43 d23 d110 d49 p171 d88 d26 p50 d141 d8 d70 p36 p37 p176 d46 p25 p126 p10 d100 p48 d159 p25 d115 p158 p100 d67 p41 d180 d162 d51 p163 p113 d16 p185 p148 d92 d118 p67 p80 p156 p103 d162 d154 p75 p176 p191 p46 d190 p36 p68 d12 p49 d106 p55 p92 d16 d149 p36 p81 p75 d137 p194 p122 d99 p141 p66 p134