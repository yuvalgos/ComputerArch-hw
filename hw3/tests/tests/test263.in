6 29 20 5
8 3 21 14
6 24 1 5
8 17 5 27
7 23 18 15
5 27 17 29
2 15 23 30
2 23 13 26
0 16 27 16
6 9 17 8
4 10 8 17
5 17 6 20
2 23 11 7
1 5 26 17
0 5 18 30
2 26 20 25
1 31 5 6
5 7 13 5
1 12 30 7
5 21 12 26
5 3 9 9
7 28 3 5
8 13 29 2
1 5 12 11
7 24 14 18
4 26 9 26
8 5 26 11
2 18 5 27
4 14 17 12
1 30 4 7
6 11 12 10
3 14 8 22
8 4 23 14
6 14 11 10
8 19 12 1
7 12 27 6
2 30 6 0
7 1 27 8
8 27 26 8
4 5 9 19
8 23 7 0
4 14 7 6
8 2 3 19
4 25 16 21
3 1 20 12
4 17 0 22
1 2 6 30
6 4 10 10
5 22 14 20
5 4 24 11
4 25 11 3
1 25 24 27
2 20 27 3
0 14 5 14
5 26 22 27
6 26 13 22
2 20 15 23
0 1 31 21
8 4 10 18
8 31 2 22
4 20 16 10
5 15 23 19
3 9 28 25
2 6 9 13
7 9 5 12
0 25 20 31
2 25 26 5
7 4 24 16
0 12 26 0
2 7 22 25
0 8 7 6
7 4 1 31
6 15 13 9
6 13 30 9
4 16 23 24
5 1 6 8
8 1 25 6
6 19 23 25
6 19 15 18
5 9 8 10
5 13 28 16
6 7 24 5
5 13 20 0
0 24 1 17
7 16 1 25
1 15 22 4
4 6 23 1
3 17 29 29
7 3 10 11
8 11 13 6
3 24 27 14
7 8 10 22
3 12 18 29
4 28 11 3
4 3 17 2
1 13 26 17
8 9 17 28
7 27 5 16
1 26 31 16
1 27 22 10
0 17 9 14
2 21 22 13
5 0 20 14
3 24 19 15
4 12 14 18
1 28 9 16
3 4 14 8
6 9 2 3
7 14 18 7
3 30 28 16
6 9 12 6
7 21 5 6
1 23 22 18
7 6 14 17
7 15 18 30
2 19 17 20
7 8 0 15
5 27 17 17
0 31 29 30
2 16 16 5
8 15 24 6
3 20 11 5
7 3 19 16
1 27 18 30
4 1 8 27
8 30 1 3
1 8 5 3
5 30 21 25
0 16 13 26
5 16 2 7
0 20 16 29
1 1 24 3
8 15 2 8
0 13 6 23
2 5 9 13
5 8 8 21
6 3 3 12
4 29 25 12
8 17 29 1
3 28 4 0
0 15 0 10
0 29 30 14
1 7 17 19
0 19 2 2
7 9 1 26
4 17 28 10
7 11 28 7
2 15 14 18
0 22 23 29
7 27 9 9
6 3 11 22
0 16 18 26
6 29 14 21
0 20 8 8
7 7 9 1
2 5 10 9
0 24 1 31
5 24 27 3
2 19 30 19
3 9 12 9
7 29 18 17
7 3 13 16
4 31 7 4
0 13 28 29
2 25 6 17
1 28 11 25
3 5 5 7
5 4 21 16
7 6 0 25
7 5 27 20
8 13 11 7
0 5 19 22
6 0 18 1
0 28 26 29
3 26 25 11
1 9 25 13
0 28 3 4
0 12 6 24
0 25 29 3
1 9 11 8
8 21 18 6
4 0 18 5
4 30 17 5
1 17 18 28
0 20 14 10
1 16 20 3
2 10 14 21
6 23 25 31
0 25 13 30
8 18 26 31
6 29 14 24
1 14 3 7
1 19 21 5
4 10 30 4
1 31 10 23
5 21 1 18
3 21 3 26
8 21 25 21
6 2 24 31
6 7 1 21
