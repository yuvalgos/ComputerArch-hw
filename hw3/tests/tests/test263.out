Reading the opcodes latency info from tests/test263.dat ... Got latency for 9 opcodes
Reading the program file tests/test263.in ... Found 200 instructions
getProgDepth()==319
getDepDepth(131)==206
getDepDepth(126)==206
getDepDepth(157)==271
getDepDepth(6)==19
getDepDepth(142)==232
getDepDepth(93)==146
getInstDeps(104)=={53,27}
getInstDeps(37)=={5,-1}
getInstDeps(4)=={-1,-1}
getDepDepth(116)==209
getDepDepth(28)==18
getDepDepth(63)==84
getDepDepth(90)==74
getDepDepth(91)==39
getInstDeps(49)=={24,30}
getInstDeps(39)=={9,34}
getDepDepth(42)==53
getDepDepth(144)==209
getInstDeps(155)=={10,144}
getDepDepth(23)==18
getDepDepth(102)==54
getDepDepth(9)==23
getDepDepth(43)==38
getInstDeps(79)=={70,10}
getInstDeps(45)=={-1,-1}
getDepDepth(162)==247
getInstDeps(112)=={48,27}
getInstDeps(85)=={48,71}
getInstDeps(132)=={46,126}
getInstDeps(26)=={25,-1}
getInstDeps(30)=={18,10}
getDepDepth(43)==38
getInstDeps(23)=={18,-1}
getInstDeps(110)=={104,86}
getInstDeps(49)=={24,30}
getDepDepth(171)==160
getInstDeps(88)=={10,30}
getInstDeps(26)=={25,-1}
getDepDepth(50)==53
getInstDeps(141)=={127,108}
getInstDeps(8)=={5,-1}
getInstDeps(70)=={69,63}
getDepDepth(36)==0
getDepDepth(37)==36
getDepDepth(176)==222
getInstDeps(46)=={-1,36}
getDepDepth(25)==40
getDepDepth(126)==206
getDepDepth(10)==23
getInstDeps(100)=={96,53}
getDepDepth(48)==26
getInstDeps(159)=={104,144}
getDepDepth(25)==40
getInstDeps(115)=={100,60}
getDepDepth(158)==155
getDepDepth(100)==180
getInstDeps(67)=={24,8}
getDepDepth(41)==15
getInstDeps(180)=={27,168}
getInstDeps(162)=={154,106}
getInstDeps(51)=={24,38}
getDepDepth(163)==104
getDepDepth(113)==182
getInstDeps(16)=={14,-1}
getDepDepth(185)==222
getDepDepth(148)==157
getInstDeps(92)=={27,0}
getInstDeps(118)=={0,109}
getDepDepth(67)==38
getDepDepth(80)==72
getDepDepth(156)==209
getDepDepth(103)==108
getInstDeps(162)=={154,106}
getInstDeps(154)=={144,131}
getDepDepth(75)==91
getDepDepth(176)==222
getDepDepth(191)==247
getDepDepth(46)==5
getInstDeps(190)=={108,157}
getDepDepth(36)==0
getDepDepth(68)==57
getInstDeps(12)=={-1,-1}
getDepDepth(49)==51
getInstDeps(106)=={53,91}
getDepDepth(55)==40
getDepDepth(92)==79
getInstDeps(16)=={14,-1}
getInstDeps(149)=={144,144}
getDepDepth(36)==0
getDepDepth(81)==52
getDepDepth(75)==91
getInstDeps(137)=={66,104}
getDepDepth(194)==275
getDepDepth(122)==187
getInstDeps(99)=={48,10}
getDepDepth(141)==155
getDepDepth(66)==57
getDepDepth(134)==203
